backend/doc/cli.md

```diff
@@ -62,34 +62,33 @@ optional arguments:
   --profile PROFILE     set user profile, valid values: users, trusted-users,
                         administrators
   --timezone TIMEZONE   set user timezone
-  --send-email          send mail to created user about user account creation
-                        (email parameter required an email notification need
-                        to be activated)
+  --send-email          send mail to created user(s) (you need to configure
+                        EMAIL-NOTIFICATION part in config file to use this
+                        feature)
```

Comment:
I have a doubt about what I write here. Can we create multiple user in same time with this command ? If no, we need to remove `(s)`.

Reply:
removed.